.github/workflows/delivery-pr-chores.yml

```diff
@@ -4,6 +4,11 @@ name: Delivery - PR Chores
 
 on:
   pull_request:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize  
```

Comment:
interestingly, this is not named `synchronized` like the rest of the activity types 🤷‍♂️ 
I initially thought that was a typo ;)